### ISSUE TITLE
spack package for man pages would be cool

### ISSUE THREAD

**Issue by samfux84** [LLNL/spack#4839]
spack provide a large repository of packages going from system tools all the way up to complex applications.

Many useful system tools are already in the builtin repo of spack.

Would it be possible to add a package that provides "man" ?

```

[sfux@develop01 ~]$ rpm -qf /usr/bin/man
man-1.6f-39.el6.x86_64
```

**Comment by alalazo** [comment]
PR welcome!

**Comment by samfux84** [comment]
solved by #4844
